StickySessions/StickySessions/View/AddName/AddNameViewController.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  File.swift
+//  StickySessions
+//
+//  Created by Patrick Steiger on 03/05/19.
+//  Copyright © 2019 CESAR. All rights reserved.
+//
+
+import UIKit
+
+class AddNameViewController : UIViewController {
+    
+    var addNameViewModel: AddNameViewModel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        addNameViewModel = AddNameViewModel()
+    }
+    
+    func loadNextViewController() {
```

Comment:
For now, we're using the logic of "performSegue". So you could do
`self.performSegue(withIdentifier: "listSessionsSegue" sender: nil)`

You'd have to have the segue on the storyboard pointing to a StoryBoard Reference which would point to the list sessions view controller